pkgs/applications/version-management/git-and-tools/gfold/default.nix

```diff
@@ -10,21 +10,18 @@
 , testers
 }:
 
-let
+rustPlatform.buildRustPackage rec {
```

Comment:
I was deliberate in avoiding recursive here when writing the derivation initially, as I consider it an anti-pattern and [I'm not alone](https://nix.dev/anti-patterns/language.html#rec-expression). I find infinite-recursion errors especially frustrating.

Reply:
Ok I'll drop my suggestions then.